### PR TITLE
Config with buildout recipe

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: .meta.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+[python]
+with-appveyor = false
+with-pypy = true
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[coverage]
+fail-under = 95
```
Comment (sallner) [inline .meta.toml:15]: Coveralls shows currently > 96%. Does it show less when run locally?
```suggestion
fail-under = 96
```

File: test_all_pythons.cfg [hunk 2]
```diff
@@ -1,29 +0,0 @@
-[buildout]
-extends = buildout.cfg
-parts += test24 test25 test26
```
Comment (sallner) [inline test_all_pythons.cfg:3]: Yeah, finally gone!

File: tox.ini [hunk 3]
```diff
@@ -1,14 +1,54 @@
...
+    COVERAGE_PROCESS_START={toxinidir}/.coveragerc
+deps =
+    coverage
+    coverage-python-version
+    zope.testrunner
+commands =
+    mkdir -p {toxinidir}/parts/htmlcov
+    coverage erase
+    coverage run -m zope.testrunner --test-path=src []
+    coverage combine
```
Comment (sallner) [inline tox.ini:52]: Does the `coverage combine` actually provide some additional value here?

**Review by sallner — APPROVED** [review]
Looks good with some minor questions.

File: .meta.toml (hunk 1 above)
Comment (icemac) [reply .meta.toml:15]: Yes, https://github.com/zopefoundation/zc.recipe.cmmi/pull/5/checks?check_run_id=1936735755 shows `95.92`

File: tox.ini (hunk 3 above)
Comment (icemac) [reply tox.ini:52]: The tests are run in separate processes by buildout. `combine` is needed to collect the coverage of the separate processes.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**